channel.go

```diff
@@ -134,6 +139,16 @@ type ChannelOptions struct {
 	ConnContext func(ctx context.Context, conn net.Conn) context.Context
 }
 
+// HandlerWithIgnore allows users to configure TChannel server such that
+// requests with method listed in NativeHanderMethods are
+// ignored by passed-in handler and natively by TChannel.
+// All other methods will be handled by passed-in handler.
+type HandlerWithIgnore struct {
+	Handler Handler
```

Comment:
since we already take a `Handler` above, we could just take `HandlerIgnores` struct, and apply any ignores on top of the existing `Handler` field?

Reply:
My original thought was to not confuse the user. If you provide this `HandlerWithIgore`, you have to provide both Handler and the to-ignore list. But looking at your comment, I think a single `[]string IgnoreMethod` should be enough. If they only "forgot" to provide this list, it will go back to the existing behavior: handle all. This is more concise. I like it.